modules/nw-networkpolicy-multitenant-isolation.adoc

```diff
@@ -68,6 +68,21 @@ $ oc apply -f <policy-name>.yaml \ <1>
 <2> Replace `<project>` with the name of the project to apply the NetworkPolicy
 object to.
 
+. If the `default` Ingress Controller configuration has the `spec.endpointPublishingStrategy: HostNetwork` value set, you must apply a label to the the `default` {product-title} namespace to allow network traffic between the Ingress Controller and the project:
+
+.. Determine if your `default` Ingress Controller uses the `HostNetwork` endpoint publishing strategy:
++
+----
+$ oc get --namespace openshift-ingress-operator ingresscontrollers/default \
```

Comment:
The correct oc command is:
[root@dhcp-41-193 FILE]# oc get --namespace openshift-ingress-operator ingresscontrollers/default --output jsonpath='{.status.endpointPublishingStrategy}'
map[loadBalancer:map[scope:External] type:LoadBalancerService]
[root@dhcp-41-193 FILE]# oc get --namespace openshift-ingress-operator ingresscontrollers/default --output jsonpath='{.status.endpointPublishingStrategy.type}'
LoadBalancerService

Reply:
Thanks @weliang1!